app/src/main/java/org/fossasia/susi/ai/forgotpassword/ForgotPasswordActivity.kt

```diff
@@ -84,8 +84,8 @@ class ForgotPasswordActivity : AppCompatActivity (), IForgotPasswordView {
     }
 
     fun showUrl() {
-        customer_server.setOnClickListener {
-            input_url.visibility = if(customer_server.isChecked) View.VISIBLE else View.GONE
+        custom_server.setOnClickListener {
```

Comment:
Change to camel case

Reply:
It is not a variable name but a resource id which is directly used as a variable using kotlinx. Should it be changed to camelCase?

Reply:
Yes, it's kotlin and java convention to use camel case

Reply:
@iamareebjamal should this change be made in a separate PR because we have to make these changes in every activity?

Reply:
OK